### PR TITLE
id migration: testworld

### PR DESCRIPTION
**Pull request by manuelpolzhofer**
- fixed testworld tests for identity branch

### CODE REVIEW

File: config/configstore/model_test.go [hunk 1]
```diff
@@ -400,7 +400,7 @@ func createMockConfig() *mockConfig {
 	c.On("GetNumWorkers").Return(2).Once()
 	c.On("GetWorkerWaitTimeMS").Return(1).Once()
 	c.On("GetEthereumNodeURL").Return("dummyNode").Once()
-	c.On("GetIdentityID").Return(utils.RandomSlice(6), nil).Once()
+	c.On("GetIdentityID").Return(utils.RandomSlice(common.AddressLength), nil).Once()
```
Comment (vimukthi-git) [inline config/configstore/model_test.go]: Please wrap `common.AddressLength` (or any other things from ethereum.common) when using outside identity. We should avoid past mistakes of leaky abstractions.

Comment (manuelpolzhofer) [reply config/configstore/model_test.go]: I had it already in mind to add a length constant to identity package. I will fix it on monday

Comment (manuelpolzhofer) [reply config/configstore/model_test.go]: Added required changes.

**Comment by vimukthi-git** [comment]
@manuelpolzhofer I think you didn't get what I meant by my comment. Lets discuss tomorrow.

**Pull request merged**